index.js

```diff
@@ -128,6 +129,9 @@ class Command extends EventEmitter {
     this._helpCommandName = 'help';
     this._helpCommandnameAndArgs = 'help [command]';
     this._helpCommandDescription = 'display help for command';
+    this._helpHeader = 'help:header';
+    this._helpBody = 'help:body';
+    this._helpFooter = 'help:footer';
```

Comment:
These can be used as literals rather than having a property for them. You were probably copying the pattern for `_helpLongFlag` but it is a property because it can be changed by the user.